v2/pkg/mirror/options.go

```diff
@@ -29,6 +29,108 @@ type ErrorShouldDisplayUsage struct {
 	Error error
 }
 
+type GlobalOptions struct {
+	LogLevel           string        // one of info, debug, trace
+	TlsVerify          bool          // Require HTTPS and verify certificates (for docker: and docker-daemon:)
+	PolicyPath         string        // Path to a signature verification policy file
+	InsecurePolicy     bool          // Use an "allow everything" signature verification policy
+	RegistriesDirPath  string        // Path to a "registries.d" registry configuration directory
+	OverrideArch       string        // Architecture to use for choosing images, instead of the runtime one
+	OverrideOS         string        // OS to use for choosing images, instead of the runtime one
+	OverrideVariant    string        // Architecture variant to use for choosing images, instead of the runtime one
+	CommandTimeout     time.Duration // Timeout for the command execution
+	RegistriesConfPath string        // Path to the "registries.conf" file
+	TmpDir             string        // Path to use for big temporary files
+	Dir                string        // working directory
+	From               string        // local storage for diskToMirror workflow
+	Port               uint16        // HTTP port used by oc-mirror's local storage instance
+	ConfigPath         string        // Path to use for imagesetconfig
+	ReleaseFrom        string        // Used for release mirroring (diskToMirror)
+	OperatorsFrom      string        // Used for operators mirroring (diskToMirror)
+	AdditionalFrom     string        // Used for additionalImages mirroring (diskToMirror)
```

Comment:
Hi @aguidirh I don't think these 3 are needed anymore since we switched to using a local registry for storage: the same imageSetConfig is used for both Mirror to disk and Disk to mirror...
But I'm not 100% sure